src/components/reusables/status-label/StatusLabel.tsx

```diff
@@ -0,0 +1,20 @@
+import "./StatusLabel.scss";
+
+import React from "react";
+
+export type LabelContent = "Pending approval" | "In Progress" | "Paid" | "Returned" | "Error" | "Cancelled";
+
+interface StatusLabelProps {
+    statusText: LabelContent;
+    styleClass: string;
+    icon: JSX.Element;

```

Comment:
```suggestion
    icon: React.ComponentType<SVGProps>;

```
I think this should work.

Reply:
hmmm, it doesnt seem to work on my branch.. will look online